src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/VSCodeHelper/VSCodeInstances.cs

```diff
@@ -123,12 +123,12 @@ public static void LoadVSCodeInstances()
                                 using var vsCodeIconBitmap = vsCodeIcon.ToBitmap();
 
                                 // workspace
-                                using var folderIcon = (Bitmap)Image.FromFile(Path.Join(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Images//folder.png"));
+                                using var folderIcon = (Bitmap)Image.FromFile(Path.Join(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Images//folder.light.png"));
```

Comment:
@jaimecbernardo This would need to be theme aware.. hardcoded it now as these were colored icons, but not sure why?

Reply:
I see, perhaps we need to do it like on other plugins. Return depending on theme, like in https://github.com/microsoft/PowerToys/commit/4d48677ad71faa4c9ddbffa39ed1314e3f1b08b4